projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -180,16 +181,24 @@ export class IgxDateRangePickerComponent extends PickerBaseDirective
     /**
      * The expected user input format and placeholder.
      *
-     * @remarks
-     * Default is `"'MM/dd/yyyy'"`
-     *
      * @example
      * ```html
      * <igx-date-range-picker inputFormat="dd/MM/yy"></igx-date-range-picker>
      * ```
      */
     @Input()
-    public override inputFormat: string;
+    public override set inputFormat(value: string) {
+        if (value) {
+            this._inputFormat = DateTimeUtil.getNumericInputFormat(this.locale, value);
```

Comment:
Similar to the base - there's no need for the `getNumericInputFormat` here, just get the default by locale and handle accordingly.